src/main/java/edu/pwr/pizzeria/model/order/dto/CustomerOrderDto.java

```diff
@@ -1,31 +1,121 @@
 package edu.pwr.pizzeria.model.order.dto;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import edu.pwr.pizzeria.model.pizza.dto.CustomPizzaDto;
-
+import edu.pwr.pizzeria.model.order.CustomerOrder;
+import edu.pwr.pizzeria.model.order.CustomerOrderStatus;
+import edu.pwr.pizzeria.model.pizza.Pizza;
+import edu.pwr.pizzeria.model.user.CustomerUser;
+import java.math.BigDecimal;
+import java.sql.Time;
+import java.util.Date;
 import java.util.List;
+import java.util.Objects;
 
-@JsonInclude(JsonInclude.Include.NON_NULL)
 public class CustomerOrderDto {
 
-    private List<Integer> pizzas;
-    private List<CustomPizzaDto> customPizzas;
+    private final Long id;
+    private final CustomerUser customerUser;
+    private final List<Pizza> pizzas;
+    private final BigDecimal total;
+    private final Date date;
+    private final Time timePassed;
+    private final CustomerOrderStatus customerOrderStatus;
+    private final String notes;
 
     @JsonCreator
-    public CustomerOrderDto(
-            @JsonProperty("pizzas") List<Integer> pizzas,
-            @JsonProperty("customPizzas") List<CustomPizzaDto> customPizzas) {
+    public CustomerOrderDto(@JsonProperty("id") Long id,
+                            @JsonProperty("customerUser")  CustomerUser customerUser,
```

Comment:
To be checked with business model and frontend - do we return here a complete user profile? Is it necessary?

Reply:
in progress